### ISSUE TITLE
app on 5656 broken

### ISSUE THREAD

**Issue by gerritbruening** [faustedition/faust-gen-html#559]
See http://dev.faustedition.net/print/faust.30#app.5656.bser

I have made this change in the wiki:
https://faustedition.uni-wuerzburg.de/wiki/index.php?title=Bibliographische_Verweise&action=historysubmit&diff=50363&oldid=50361

Or will this work tomorrow, as the wiki bibliography will be pulled over night?

**Comment by gerritbruening** [comment]
Thanks, now the entry works.
On this occasion I notice, however, that C.1 41 and C.3 41 are rendered with superscirpt. Is this intended?

**Comment by thvitt** [comment]
Consequence of the print rendering + the handling of the 'fake' sigils via bibliography

**Comment by gerritbruening** [comment]
> Is this a problem?

Not really, it may look inconsistent for those users who know that C.1 and C¹ refer to the same edition.

**Comment by dietmarpravida** [comment]
que diable vais-je faire dans cette galère?

**Comment by gerritbruening** [comment]
C'est une question d'une importance fondamentale!
